docs/_ext/powershell_lexer.py

```diff
@@ -0,0 +1,5 @@
+from pygments.lexers import get_lexer_by_name  # refer LEXERS
```

Comment:
not sure of the utility of this comment

Reply:
Copied from an example, fixed in 96aa0b05d9f83941cd1a072eeeaa3d5badc77997